test/dlc_tests/sanity/test_safety_check.py

```diff
@@ -128,10 +128,10 @@ def _get_latest_package_version(package):
 @pytest.mark.model("N/A")
 @pytest.mark.canary("Run safety tests regularly on production images")
 @pytest.mark.skipif(not is_dlc_cicd_context(), reason="Skipping test because it is not running in dlc cicd infra")
-@pytest.mark.skipif(not (is_mainline_context() or (is_canary_context() and is_time_for_canary_safety_scan())),
-                    reason="Skipping the test to decrease the number of calls to the Safety Check DB. "
-                           "Test will be executed in the 'mainline' pipeline and canaries pipeline."
-                    )
+# @pytest.mark.skipif(not (is_mainline_context() or (is_canary_context() and is_time_for_canary_safety_scan())),
+#                     reason="Skipping the test to decrease the number of calls to the Safety Check DB. "
+#                            "Test will be executed in the 'mainline' pipeline and canaries pipeline."
+#                     )
```

Comment:
To be reverted when PR is soft-approved.

Reply:
Will this skip safety tests in PR?